README.md

```diff
@@ -1,2 +1,156 @@
+| options  | Specifies the PouchDB database `options`. <br>Example<br>- SQLite : {adapter: 'websql'}<br>- LevelDB: {adapter: 'leveldb'} or {}<br>- CouchDB: {} <br>`default(SQLite): {adapter: 'websql'}`|
+
+Reference: [PouchDB Create a database options](https://pouchdb.com/api.html#create_database)
+
+## Data Model
+
+- This plugin uses a PouchDB database for all context scope.
+- The NodeSQLite adapter is added to the PouchDB adapter. You can save the data in SQLite3.
+- You can also specify saving to a database (LevelDB, CouchDB) that PouchDB supports as standard.
+- This plugin saves a JSON object of keys and values in a document for each scope.
+  - The keys of `global context` will be id with `global` .
+  - The keys of `flow context` will be id with `<id of the flow>` .
+  - The keys of `node context` will be id with `<id of the node>` .
+  - Context data is stored in `doc.data` as a document for each scope.
+
+Structure of data stored in PouchDB:
+```json
+{
+    "total_rows": 3,
+    "offset": 0,
+    "rows": [
+        {
+            "id": "2052fca8.312154:a77d79a4.d1a908",
+            "key": "2052fca8.312154:a77d79a4.d1a908",
+            "value": {
+                "rev": "6-55e0513ffba64a8b8efec1ba8e43c90f"
+            },
+            "doc": {
+                "data": {
+                    "NODE-KEY-1": "NODE-DATA-1",
+                    "NODE-KEY-2": "NODE-DATA-2"
+                },
+                "_id": "2052fca8.312154:a77d79a4.d1a908",
+                "_rev": "6-55e0513ffba64a8b8efec1ba8e43c90f"
+            }
+        },
+        {
+            "id": "a77d79a4.d1a908",
+            "key": "a77d79a4.d1a908",
+            "value": {
+                "rev": "61-2c2a457388db1c3859b79e4bb62e9375"
+            },
+            "doc": {
+                "data": {
+                    "FLOW-KEY-1": "FLOW-DATA-1",
+                    "FLOW-KEY-2": "FLOW-DATA-2",
+                },
+                "_id": "a77d79a4.d1a908",
+                "_rev": "61-2c2a457388db1c3859b79e4bb62e9375"
+            }
+        },
+        {
+            "id": "global",
+            "key": "global",
+            "value": {
+                "rev": "73-ee260387e51ae20132076ccc83957600"
+            },
+            "doc": {
+                "data": {
+                    "GLOBAL-KEY-1": "GLOBAL-DATA-1",
+                    "GLOBAL-KEY-2": "GLOBAL-DATA-2",
+                },
+                "_id": "global",
+                "_rev": "73-ee260387e51ae20132076ccc83957600"
+            }
+        }
+    ]
+}
+```
+
+## Data Structure
+
+- Data is saved in the JSON object format supported by PouchDB. The plugin does not convert JSON data to a string for storage.
+
+Code example that references database data :
```

Comment:
The code example is supposed to be executed externally, and can also be run in the function node.
When executing in the function node, set the functionGlobalContext in setting.js so that PouchDB can be used in the function node.

Reply:
maybe worth adding that as a "how-to" in the readme. (probably more applicable to the events example though).